code/_onclick/human.dm

```diff
@@ -86,6 +86,7 @@
 	return HANDLE_CLICK_PASS_THRU
 
 /atom/proc/attack_hand(mob/user)
+	SEND_SIGNAL(src, COMSIG_ATOM_ATTACK_HAND, user)
```

Comment:
bit of a problem with this is it's unreliable since not everything calls parent

Reply:
This is unfortunately a bit of an issue that's hard to solve (see atom/attackby() and etc.) and is out of scope to fix in this PR

Reply:
Could perhaps just toss in a comment about it instead, so future coders don't figure that out the hard way

Reply:
done